glib/src/variant.rs

```diff
@@ -244,15 +244,27 @@ impl Variant {
     ///
     /// * if `self` is not a container type.
     /// * if given `index` is larger than number of children.
-    #[doc(alias = "get_child_value")]
-    #[doc(alias = "g_variant_get_child_value")]
     pub fn child_value(&self, index: usize) -> Variant {
-        assert!(index < self.n_children());
         assert!(self.is_container());
+        assert!(index < self.n_children());
```

Comment:
Self-note: this re-ordering is happening in order to align with the other conditionals, and also because `n_children()` does panic on non-container input.